h/accounts/test/views_test.py

```diff
@@ -413,8 +426,7 @@ def test_forgot_password_redirects_on_success(authn_policy):
 
 @pytest.mark.usefixtures('routes_mapper')
 def test_forgot_password_form_redirects_when_logged_in(authn_policy):
-    request = DummyRequest()
-    authn_policy.authenticated_userid.return_value = "acct:jane@doe.org"
+    request = DummyRequest(authenticated_userid="acct:jane@doe.org")
```

Comment:
Why is this change needed? If the `authn_policy` fixture isn't working somehow, then let's fix it. But it should definitely work here...